godtools/App/Share/SwiftUI Views/LessonCardView/ViewModels/LessonCardViewModel.swift

```diff
@@ -130,6 +115,15 @@ extension LessonCardViewModel {
                 }
             }
         }
+        
+        getBannerImageUseCase.getBannerImagePublisher(for: resource.attrBanner)
```

Comment:
Hey @rachaelblue for this check out the assign() method https://developer.apple.com/documentation/combine/just/assign(to:on:).

You should be able to do something like this.

```
getBannerImageUseCase.getBannerImagePublisher(for: resource.attrBanner)
            .receive(on: DispatchQueue.main)
            .assign(to: \.bannerImage, on: self)
            .store(in: &cancellables)
```

Reply:
Oh cool!  That cleans it up nicely.  Thanks!